cancellation/index.md

```diff
@@ -23,9 +23,11 @@ Our legal responsibility is to account owners, which means we cannot cancel an a
 
 ## What happens when you cancel?
 
-You won’t be able to access your account once you cancel, so make sure you download everything you want to keep beforehand.
+In the case of HEY, if you paid for your account and you cancel your subscription, you can
```

Comment:
Pardon me chiming in here - it looks like this sentence was cut off? It looks to be missing what you can do if you paid and cancelled.

Reply:
Oh, yes, totally! Thanks @fm! I'll fix 😊

Reply:
Removed that sentence, it's covered in the next paragraph. I just forgot to delete it once I wrote this in a different way.